### DISCUSSION TITLE
Sketch component for the Select with option groups

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1013] [Q&A] [answered]
I'm looking for a Sketch component for the Select with option groups. I couldn't locate it so I used the existing **Options list** and changed typography for grouping but I'm sure there is a better way. 
<img width="364" alt="Screen Shot 2020-12-08 at 3 22 38 PM" src="https://user-images.githubusercontent.com/67388401/101553605-d56d6e00-3969-11eb-9699-1dbf74b7b955.png">
Please let me know what should I use.  Thanks!

**Comment by serifluous** [comment, marked as answer]
This is great if it works well enough to communicate with your team! We didn't create a symbol for option groups for Select because we didn't imagine too many people would be using it. It's good to see that you are! We might create the variant if we see even more usage.
